presto-product-tests/src/main/java/io/prestosql/tests/hive/TestCsv.java

```diff
@@ -164,6 +216,103 @@ private void testCreatePartitionedCsvTableAs(String tableName, String additional
         query("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateCsvFileSkipHeader()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_create_csvfile_skip_header");
+        onPresto().executeQuery("" +
+                "CREATE TABLE test_create_csvfile_skip_header" +
+                " (column1 varchar, column2 varchar, column3 varchar) " +
+                "WITH ( " +
+                "   format = 'CSV', " +
+                "   external_location = 'hdfs://hadoop-master:9000/user/hive/warehouse/TestCsvFileHiveTable/csv_table_with_header', " +
+                "   csv_separator= ','," +
+                "   skip_header_line_count = 1 " +
+                ")");
+        assertThat(query("SELECT * FROM test_create_csvfile_skip_header")).containsOnly(row("value1", "value2", "value3"));
+        onHive().executeQuery("DROP TABLE test_create_csvfile_skip_header");
+    }
+
+    @Test
+    public void testCreateCsvFileSkipFooter()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_create_csvfile_skip_footer");
+        onPresto().executeQuery("" +
+                "CREATE TABLE test_create_csvfile_skip_footer" +
+                " (column1 varchar, column2 varchar, column3 varchar) " +
+                "WITH ( " +
+                "   format = 'CSV', " +
+                "   external_location = 'hdfs://hadoop-master:9000/user/hive/warehouse/TestCsvFileHiveTable/csv_table_with_footer', " +
+                "   csv_separator= ','," +
+                "   skip_footer_line_count = 1 " +
+                ")");
+        assertThat(query("SELECT * FROM test_create_csvfile_skip_footer")).containsOnly(row("value1", "value2", "value3"));
+        onHive().executeQuery("DROP TABLE test_create_csvfile_skip_footer");
+    }
+
+    @Test
+    public void testCreateCsvFileSkipHeaderFooter()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_create_csvfile_skip_header_footer");
+        onPresto().executeQuery("" +
+                "CREATE TABLE test_create_csvfile_skip_header_footer" +
+                " (column1 varchar, column2 varchar, column3 varchar) " +
+                "WITH ( " +
+                "   format = 'CSV', " +
+                "   external_location = 'hdfs://hadoop-master:9000/user/hive/warehouse/TestCsvFileHiveTable/csv_table_with_header_footer', " +
+                "   csv_separator= ','," +
+                "   skip_header_line_count = 1, " +
+                "   skip_footer_line_count = 1 " +
+                ")");
+        assertThat(query("SELECT * FROM test_create_csvfile_skip_header_footer")).containsExactly(row("value1", "value2", "value3"));
+        onHive().executeQuery("DROP TABLE test_create_csvfile_skip_header_footer");
+    }
+
+    @Test
+    public void testInsertCsvSkipHeader()
```

Comment:
can you merge test inserts with create table tests. Tempto tests are a bit heavy.